src/hooks/userGetUserData.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { getUserData } from '../utils/userData.ts';
+
+export const useGetUserData = () => {
+  const [loading, setLoading] = useState(true);
+  const [userData, setUserData] = useState([]);
+  const [err, setErr] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const result = await getUserData(
+          'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VyX2lkIjoxMDEsImV4cCI6MTcxMTY2MjY3OH0.yJEfuBfNQVsgvfNaXwb0CfzgH_UUOUHowO5iLnk_sNo',
+          1
```

Comment:
The Token will change every x amount of time. Do you have any other idea for storing it somewhere instead of having this hardcoded?

Reply:
Implement a JS Object (dictionary) in local memory for now.

But create a separate file and allocate the dictionary there, so in the future we can easily change the local memory dictionary for something like Redis or Async Storage to have some persistency with the tokens.